guides/common/attributes.adoc

```diff
@@ -15,6 +15,9 @@
 
 // Use current RH Satellite release version for links to access.redhat.com
 :AccessRedHatComVersion: 6.10
+// Red Hat sometimes prefers the multi page (as opposed to html-single)
+// The URL structure is different and there's no upstream equivalent.
+:MultiBaseURL: https://access.redhat.com/documentation/en-us/red_hat_satellite/{AccessRedHatComVersion}/html/
```

Comment:
Can you move this few lines below to the section where we also define the `BaseURL`? I suggest also to override it for non-satellite context to be actually `BaseURL` so it works seamlessly.

Reply:
> Can you move this few lines below to the section where we also define the BaseURL?

Sure

> I suggest also to override it for non-satellite context to be actually BaseURL so it works seamlessly.

I don't know which value.

Reply:
My thinking was to set `MultiBaseURL` to something like `https://docs.theforeman.org/{Version}/` so it is rendered correctly in non-Satellite contexts too.

Reply:
I think that will result into 404s.

From the example in https://github.com/theforeman/foreman-documentation/pull/856#discussion_r773898246:
```
{MultiBaseURL}/upgrading_and_updating_red_hat_satellite/upgrading_red_hat_satellite
```

This would then result in:
```
https://docs.theforeman.org/{Version}/upgrading_and_updating_red_hat_satellite/upgrading_red_hat_satellite
```

However, we only have:
```
https://docs.theforeman.org/{Version}/upgrading_and_updating_red_hat_satellite/index-foreman-el.html
https://docs.theforeman.org/{Version}/upgrading_and_updating_red_hat_satellite/index-foreman-deb.html
https://docs.theforeman.org/{Version}/upgrading_and_updating_red_hat_satellite/index-katello.html
```

We also don't have rewrite rules so it's not like we can configure Apache to solve it for us. We already use fragments so can't solve it there either.